vdev/src/testing/config.rs

```diff
@@ -73,6 +73,12 @@ pub struct IntegrationTestConfig {
     /// Configuration specific to the compose services.
     #[serde(default)]
     pub runner: IntegrationRunnerConfig,
+
+    pub features: Vec<String>,
```

Comment:
It might be just as simple to use a plain `String` here.

Reply:
I opted to keep the original Vector of strings, but will change to String if it is better to do so.